makefiles/Makefile.docs-mongoid

```diff
@@ -70,62 +70,4 @@ next-gen-stage: ## Host online for review
 		mut-publish public ${BUCKET} --prefix="${MUT_PREFIX}" --stage ${ARGS}; \
 		echo "Hosted at ${URL}/${MUT_PREFIX}/${USER}/${GIT_BRANCH}/"; \
-	fi
-
-########################
-# GIZA MAKEFILE COMMANDS
-########################
-
-html:
-	giza make html
-
-publish: ## Build docs locally
-# !!! DOES NOT PUT STUFF ONTO THE INTERNET !!!
-# Builds the artifacts that you will deploy with other targets.
-# Also builds an HTML rendering for local viewing.
-#
-# If master branch, giza creates any symlinks needed per integrations.yaml
-# If master, giza creates any redirects per build_conf.yaml / htaccess.yaml
-
-	# get the latest for mongoid/docs git submodule:
-	git submodule update --remote --init
-	# rsync the docs source from the submodule to the source/ directory:
-	rsync -a --delete mongoid/docs/ source/
-	# build the publish artefacts using giza:
-	giza make publish
-	if [ ${GIT_BRANCH} = master ]; then mut-redirects config/redirects -o build/public/.htaccess; fi
-
-stage: ## Host online for review
-# !!! DEPLOY HTML TO STAGING WEBSITE !!!
-# Deploys the HTML from the build/<branch> directory to the BUCKET.
-#
-# mut-publish
-#       build/${GIT_BRANCH}/html  directory from which to grab the HTML to deploy to the BUCKET
-#       	BUCKET            Amazon s3 bucket used for the staging site
-#         --prefix=${PROJECT}      ${PROJECT} is the folder in the staging bucket in which to place the HTML
-#      --stage                    indicates that mut-publish should publish to the staging bucket (not the prod bucket)
-#      if ${ARGS}, then additonal arguments
-
-	mut-publish build/${GIT_BRANCH}/html ${BUCKET} --prefix=${PROJECT} --stage ${ARGS}
-	@echo "Hosted at ${URL}/${PROJECT}/${USER}/${GIT_BRANCH}/index.html"
-
-
-
-deploy: publish ## Deploy to the production bucket
-# !!! DEPLOY BUILD ARTIFACTS TO PRODUCTION !!!
-# Deploys the build artifacts from the build/public directory to the PRODUCTION_BUCKET,
-# first doing a 'dry run' that lists all of the files that will be uploaded/deleted/etc.
-# and then asks you to press a key to perform the procedure.
-#
-# deploy: publish 	the :publish indicates that the publish target must happen before the deploy
-# mut-publish
-#      build/public         directory from which to grab the build artifacts to deploy to the PRODUCTION_BUCKET
-#      			BUCKET    Amazon s3 bucket that holds the production docs
-#      --prefix=${PROJECT}   ${PROJECT} is the folder in the prod. bucket in which to place the artifacts
-#      --deploy             indicates that mut-publish should publish to the production bucket (not the staging bucket)
-#      --verbose            prints out a detail of what files are being uploaded/deleted/etc.
-#      --dry-run            instructs mut-publish to do everything *except* actually put stuff on the internet.
-#      if ${ARGS}, then additonal arguments
-	mut-publish build/public ${BUCKET} --prefix=${PROJECT} --deploy ${ARGS}
-
-	@echo "Hosted at ${URL}/${PROJECT}/index.html"
+	fi
```

Comment:
There's no terminal newline on this one and Makefiles are weird about things, so please put one there in case it matters.

Reply:
Will do!

Reply:
Should be fixed, let me know if that's not what you had in mind

Reply:
Sorry - I should have been more clear. We don't need a terminal `\`, we just need like literally to hit enter at the end of the `fi` line to move the cursor to the next row (so that little red negative circle disappears).